### ISSUE TITLE
Not drawing when editor is moved

### ISSUE THREAD

**Issue by ssr765** [robertrosman/vue-paint#11]
When the editor changes its position, it only accepts inputs from its previous location. As shown in the video, you need to "draw" where the editor was previously to make it accept drawing in its new position.

https://github.com/robertrosman/vue-paint/assets/64972770/8b146028-7aab-421f-b22e-f6c031282cc4

**Comment by achtelik** [comment]
As additional information. I could only reproduce this with "position: absolute".
If you only use the margin it still works like expected.

**Comment by ssr765** [comment]
Yup, I was working on a project that you can drag the "canvas" around the screen, so didn't try with margin.
